src/ensq_channel.erl

```diff
@@ -122,7 +122,7 @@ loop(State) ->
                              %% We don't want to ask for a propper new RC every time
                              %% this keeps the laod of the flow manager by guessing
```

Comment:
Line 123 is too long:                              %% this keeps the laod of the flow manager by guessing.